src/v0/destinations/criteo_audience/config.js

```diff
@@ -0,0 +1,10 @@
+const BASE_ENDPOINT = "https://api.criteo.com/2022-10/";
```

Comment:
Should we think of making this version env or something ? @krishna2020 @koladilip

Reply:
Accommodating version

Reply:
Accomodated